.eslintrc.json

```diff
@@ -15,6 +15,7 @@
   "globals": {
     "AutocompleteElement": "readonly"
   },
+  "ignorePatterns": ["examples/dist"],
```

Comment:
Looks like there is a `eslintIgnore` in the `package.json` as well.  Let's move both in here